### DISCUSSION TITLE
Resume GitHub releases and changelog

### DISCUSSION THREAD

**Discussion by shishkin** [solidjs/solid#1412] [Ideas]
Hi! I've noticed that releases and changelog were stopped after 1.6.0. Are there plans to resume it? It would be nice to have as some tools like renovate use them to show in the update PRs.

**Comment by ryansolid** [comment]
I only ever did this for major/minor releases historically. Just took time to write up so I never bothered with patches. So haven't changed anything there. However we recently introduced changesets which make the process of documenting patch releases a bit easier. Each package now has a detailed changelog.

**Reply by shishkin** [reply]
Thanks! I didn't realize changelogs are now separate per package, e.g. in https://github.com/solidjs/solid/blob/main/packages/solid/CHANGELOG.md. Maybe renovate is then not able to find them and needs to get smarter for monorepo projects.
